config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
On Wed, May 11, 2016 at 03:12:24AM -0700, James Hunt wrote:
> I think we might want to consider adding ``RuntimePath``. Currently,
> ``dockerd`` will launch ``containerd`` and ``containerd`` will
> launch the runtime configured when it was started.
> 
> But this implies that if a single host is to support multiple
> runtimes, *you would need two instances of* ``containerd``:

I acknowledge the problem, but think that this is a dockerd ↔
containerd issue [1].  There's no need to put this in runtime-spec if
the containerd API changes to support:

  launch(pathToBundle, runtimePath, runtimeArgs)

[1]: https://github.com/opencontainers/runtime-spec/pull/405/files/6194f0b24749bedaf3f384787b73cf1c9f899f4b#r61617026